src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFeaturedVideoSound.java

```diff
@@ -42,7 +42,7 @@ public void adsFeaturedVideoWithoutSoundMercury() {
   }
 
   private void verifyFeaturedVideoWithSound(String pageUrl) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyFeaturedVideoWithSound" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)